scripts/keybinding.sh

```diff
@@ -9,7 +9,5 @@ target=$(tmux list-keys | sed '1s/^/[cancel]\n/' | eval "$TMUX_FZF_BIN $TMUX_FZF
 [[ "$target" == "[cancel]" || -z "$target" ]] && exit
 if [[ -n $(echo "$target" | grep -o "copy-mode") && -z $(echo "$target" | grep -o "prefix") ]]; then
     tmux copy-mode
-    echo "$target" | sed -E 's/^.{46}//g' | xargs tmux
-else
-    echo "$target" | sed -E 's/^.{46}//g' | xargs tmux
 fi
+echo "$target" | cut -c 46- | xargs tmux
```

Comment:
Two problems.

First of all, the logic of `sed -E 's/^.{46}//g'` is cutting the first 46 characters, starting from the 47th. But `cut -c 46-` starts from the 46th.

The second problem is that, I cut the first 46 characters because the output of `tmux list-keys` contains valid commands from the 47th character at the time I wrote this piece of code, but now the valid commands no longer starts from the 47th character. That's a horrible mess, and it's my mistake :(

We should use awk here.

Reply:
Good catch. It would have added some leading space.

Here are my experiments:

``` bash
$ tmux list-keys | head -3
bind-key    -T copy-mode    C-Space              send-keys -X begin-selection
bind-key    -T copy-mode    C-a                  send-keys -X start-of-line
bind-key    -T copy-mode    C-b                  send-keys -X cursor-left

$ tmux list-keys | cut -c 46- | head -3
    send-keys -X begin-selection
    send-keys -X start-of-line
    send-keys -X cursor-left

$ tmux list-keys | cut -c 50- | head -3
send-keys -X begin-selection
send-keys -X start-of-line
send-keys -X cursor-left

# source https://stackoverflow.com/a/2961711
$ tmux list-keys | head -3 | awk '{for (i = 5; i <= NF; i++) {printf "%s ", $i}; printf "\n"}'
send-keys -X begin-selection
send-keys -X start-of-line
send-keys -X cursor-left
```

Reply:
The awk implementation looks good. Could you modify your code to use awk? Or I'll merge your code and implement it.

Reply:
I've come up with other alternatives:

``` bash
# source https://stackoverflow.com/a/2961994
$ tmux list-keys | head -3 | awk '{ $1=$2=$3=$4=""; print $0 }' | sed 's/^ *//'
send-keys -X begin-selection
send-keys -X start-of-line
send-keys -X cursor-left

# the problem of this option is when consecutive spaces appear in the last column (the command)
$ tmux list-keys | head -3 | sed -E 's/  +/ /g' | cut -d\  -f 5-
send-keys -X begin-selection
send-keys -X start-of-line
send-keys -X cursor-left

```

IMO the second awk option is the most readable and reliable